HW0/P6/P6B.py

```diff
@@ -16,16 +16,24 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [a**b for a,b in zip([10]*10,range(-10,1))]
 
     for t in wait_time:
+        print "burnTime", t
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
-
+        serialTime = time.time()
+        for i in xrange(N):
+            burnTime(t)
+        serialTime = time.time() - serialTime
+        
+        parallelTime = time.time()
+        pool.map(burnTime, [t]*N)
+        parallelTime = time.time() - parallelTime
+        
+        print "serialTime", serialTime, "parallelTime", parallelTime
         # Compute the ratio of these times
-        # ratio.append(serialTime/parallelTime)
+        ratio.append(serialTime/parallelTime)
 
     # Plot the results
     plt.plot(wait_time, ratio, '-ob')
```

Comment:
I think the question explicitly asks to add the plot for this problem and name it P6.png. I would also suggest adding the graphs for the previous problems even though the problem may not explicitly ask for them. Just makes it easier for us to compare.